guides/plugins/apps/storefront/apps-as-theme.md

```diff
@@ -24,11 +11,10 @@ So the folder structure of a theme may look like this:
       └── manifest.xml
 ```
 
-### Themes vs. "ordinary" apps
+## Themes vs. "ordinary" apps
 
 If your app provides a `theme.json` file it is considered to be a theme. All the changes you do to the storefront's appearance inside your theme, will be visible only if your theme is assigned to the storefront. In contrast if you don't provide a `theme.json` file, your app is a "ordinary" app and the changes of your app will be applied to all saleschannels automatically, as long as your app is active.
```

Comment:
The term `sales channels` is written lower case because it is not a name. Storefront is written in upper case because we are talking about our product (the Storefront).